product_supplierinfo_discount/README.rst

```diff
@@ -47,6 +47,7 @@ Contributors
 ------------
 
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Ignacio Ibeas <ignacio@acysos.com>
```

Comment:
1.Go to Purchase > Products, open one product,
2. edit or add a record on the *Suppliers* section of the *Procurements*. 
3. In the prices section, a new column called *Discount (%)* has been 
added where you can input  the desired discount for that quantity.
4.  When you make a purchase order for that supplier and that product, the discount
 will be automatically set.